for-developers/sdk/integration/messaging.mdx

```diff
@@ -0,0 +1,247 @@
+```ts
+// integrations.definition.ts
+import { z, IntegrationDefinition } from '@botpress/sdk'
+
+export default new IntegrationDefinition({
+  name: "webhook-message",
+  version: '0.0.1',
+  configuration: {
+    schema: z.object({
+      webhookUrl: z.string().describe('The url to post the bot answers to.'),
+    }),
+  },
+  channels: {
+    webhook: {
+      conversation: {
+        tags: {
+          id: { title: 'Conversation ID', description: 'The ID of the conversation in the external service' },
+        },
+      },
+      messages: {
+        // this channel only supports text messages
+        text: {
+          schema: z.object({
+            text: z.string(),
+          }),
+        },
+      },
+    },
+  },
+  user: {
+    tags: {
+      id: { title: 'User ID', description: 'The ID of the user in the external service' },
+    },
+  },
+})
+```
+
+Let's break down this data structure:
+
+- **`configuration`**: Defines the settings bot developpers must provide when installing the integration in the Studio. The configuration schema is used to display a configuration form. Here, only a `webhookUrl` is required to receive outgoing messages.
+- **`channels`**: Specifies supported messaging channels. This example uses a single `webhook` channel, but you can define multiple. For example, Github allows conversing with your both on issues and pull requests. Each channel can have its own set of tags and message types.
+- **`conversation.tags`**: Defines available tags for the conversation. In this case, we are defining a single tag called `id` that identifies the conversation in the external service. This allows binding the botpress conversation to the external service conversation. When sending outgoing message response, this tag will be used to properly route the message to the external service.
+- **`user.tags`**: Defines available tags for the user. Here, we define a single tag called `id` that identifies the user in the external service. This allows binding the botpress user to the external service user.
+- **`messages`**: Defines available message types. In this case, we are defining a single message type called `text` that contains a single field called `text`. This allows sending text messages to the external service. You could define multiple message types, for example, `image`, `video`, etc. The SDK exports a record of default message types that are supported in most messaging channels.
+
+### Channel Implementation
+
+The integration implementation looks something like this:
+
+{/* TODO(driftable): This script is subject to changes and should be auto-generated */}
+```ts
+import * as sdk from '@botpress/sdk'
+import axios from 'axios'
+import * as bp from '.botpress'
+
+export default new bp.Integration({
+  register: async () => {},
+  unregister: async () => {},
+  actions: {},
+  channels: {
+    webhook: {
+      messages: {
+        text: async (props) => {
+          /**
```

Comment:
Issue: The comment using `/*` doesn't render properly. We should use`//` instead.

<img width="817" alt="image" src="https://github.com/user-attachments/assets/523a2e19-5084-430a-97db-329ea3213e8a" />